testfixtures/tests/test_comparison.py

```diff
@@ -427,6 +427,73 @@ def test_object_not_strict(self):
             AClass(1, 2),
             )
 
+    def test_property_equal_strict(self):
+        self.run_property_equal_test(strict=True)
+
+    def test_property_equal_not_strict(self):
+        self.run_property_equal_test(strict=False)
+
+    def run_property_equal_test(self, strict):
+        class SomeClass(object):
+            @property
+            def prop(self):
+                return 1
+
+        self.assertEqual(
+            C(SomeClass, prop=1, strict=strict),
+            SomeClass()
+        )
+
+    def test_property_not_equal_strict(self):
+        self.run_property_not_equal_test(strict=True)
+
+    def test_property_not_equal_not_strict(self):
+        self.run_property_not_equal_test(strict=False)
+
+    def run_property_not_equal_test(self, strict):
+        class SomeClass(object):
+            @property
+            def prop(self):
+                return 1
+
+        self.assertNotEqual(
+            C(SomeClass, prop=2, strict=strict),
+            SomeClass()
+        )
```

Comment:
Not the "not equal" tests, please can you also make assertions about the repr of the `Comparison`.
The feedback from this is an important part of how these objects work in terms of giving good feedback when things aren't as expected. 

eg: https://github.com/Simplistix/testfixtures/blob/master/testfixtures/tests/test_comparison.py#L203-L214

Reply:
I did consider this and decided to omit assertions on repr because I figured it would be over-testing (I haven't changed anything to do with repr construction) but am equally happy to add these.

On reflection, I guess it's conceivable that the repr construction wouldn't work correctly for properties or (more likely) menthods, so seems a good idea to have this.